### PR TITLE
tests: refactor tests with authentication

### PR DESCRIPTION
**Pull request by BettyB979**

### CODE REVIEW

File: src/main/java/fr/insee/survey/datacollectionmanagement/config/auth/security/GrantedAuthorityConverter.java [hunk 1]
```diff
@@ -13,27 +15,39 @@
...
+                .filter(grantedRoles::containsKey)
+                .map(grantedRoles::get)
                 .collect(Collectors.toCollection(ArrayList::new));
     }
+
+    private void fillGrantedRoles(List<String> listRoles, AuthorityRoleEnum roleEnum) {
+
+        for (String role : listRoles) {
+            this.grantedRoles.put(role,
+                    new SimpleGrantedAuthority(roleEnum.securityRole()));
```
Comment (davdarras) [inline src/main/java/fr/insee/survey/datacollectionmanagement/config/auth/security/GrantedAuthorityConverter.java:48]: what if same config role for multiple app roles ?

File: src/main/java/fr/insee/survey/datacollectionmanagement/contact/controller/AddressController.java [hunk 2]
```diff
@@ -30,9 +29,6 @@
 import java.util.List;
 
 @RestController
-@PreAuthorize("@AuthorizeMethodDecider.isInternalUser() "
```
Comment (davdarras) [inline src/main/java/fr/insee/survey/datacollectionmanagement/contact/controller/AddressController.java:33]: normal to not have preauthorize in new version ?

File: src/main/java/fr/insee/survey/datacollectionmanagement/query/controller/QuestioningController.java [hunk 3]
```diff
@@ -98,9 +97,6 @@ public ResponseEntity<?> getQuestioningsBySurveyUnit(@PathVariable("id") String
 
     }
 
-    @PreAuthorize("@AuthorizeMethodDecider.isInternalUser() "
```
Comment (davdarras) [inline src/main/java/fr/insee/survey/datacollectionmanagement/query/controller/QuestioningController.java:101]: no preauthorize in new version ?

Comment (BettyB979) [reply src/main/java/fr/insee/survey/datacollectionmanagement/query/controller/QuestioningController.java:101]: @PreAuthorize(AuthorityPrivileges.HAS_MANAGEMENT_PRIVILEGES) already on top of controller

File: src/main/java/fr/insee/survey/datacollectionmanagement/contact/controller/AddressController.java (hunk 2 above)
Comment (BettyB979) [reply src/main/java/fr/insee/survey/datacollectionmanagement/contact/controller/AddressController.java:33]: oups

**Pull request merged**